app/code/community/Bolt/Boltpay/controllers/ApiController.php

```diff
@@ -238,6 +238,40 @@ protected function getCaptureAmount($transaction) {
         return null;
     }
 
+    /**
+     * Creates the success url for Bolt to forward the customer browser to upon transaction authorization
+     *
+     * @param Mage_Sales_Model_Order    $order
+     * @param Mage_Sales_Model_Quote    $immutableQuoteId
+     *
+     * @return string   The URL for which Bolt is to forward the browser.  It contains variables normally
+     *                  stored as session values as URL parameter
+     *
+     * @throws Mage_Core_Model_Store_Exception  if for any reason the store can not be found to generate the URL
+     */
+    private function createSuccessUrl($order, $immutableQuoteId) {
+        /* @var Mage_Sales_Model_Quote $immutableQuote */
+        $immutableQuote = Mage::getModel('sales/quote')->loadByIdWithoutStore($immutableQuoteId);
+        $recurringPaymentProfiles = $immutableQuote->collectTotals()->prepareRecurringPaymentProfiles();
+        $successUrlPath = $this->boltHelper()->getMagentoUrl(Mage::getStoreConfig('payment/boltpay/successpage')) ?: '/';
```

Comment:
Might be able to clean this up a little by passing the query string parameters as a second argument to getMagentoUrl instead of concatenating the string like this? See the example here https://github.com/BoltApp/bolt-magento1/pull/348/files#diff-8a9f070472f68b882d410cebf95f5bd3R213

Not that important. Just saw it here and on line 260 and thought I'd point it out.

Otherwise LGTM

Reply:
Thanks!  I'll update it.